drivers/builtin/include/mbedtls/config_adjust_legacy_crypto.h

```diff
@@ -378,4 +378,15 @@
 #define MBEDTLS_SSL_HAVE_AEAD
 #endif
 
+/* Deduce MBEDTLS_CTR_DRBG_USE_128_BIT_KEY from MBEDTLS_PSA_CRYPTO_RNG_STRENGTH */
```

Comment:
Ok, then `MBEDTLS_CTR_DRBG_USE_128_BIT_KEY` should no longer be a configuration option. I guess this needs to wait until https://github.com/Mbed-TLS/mbedtls/pull/10292 is merged?